logical/testing/testing.go

```diff
@@ -135,7 +135,7 @@ func Test(tt TestT, c TestCase) {
 	}
 
 	// Create an in-memory Vault core
-	logger := logformat.NewVaultLogger(log.LevelTrace)
+	logger := logformat.NewVaultLogger(log.Debug)
```

Comment:
This should be switched back to `Trace`

Reply:
@briankassouf yes to all of your comments, heading through them and doing them.